guac-restAPI-integration.md

```diff
@@ -0,0 +1,123 @@
+
+---
+
+# GUAC REST API Documentation
+
+**Version:** Experimental (v0.4.0 - v0.6.0)  
```

Comment:
The "- v0.6.0" doesn't seem to make sense here. We still have the rest API in more recent versions, so what is this trying to communicate?

I'm not sure we want to mention the version at all, since we don't have multiple supported versions and this just adds more places to update documentation for every release. Putting "introduced in version ..." would be reasonable, though.

Reply:
So which versions would you like me to mention ? Or i delete the segment ?

Reply:
Deleting, as you did, seems like a good approach.